homework.md

```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the apps/websites
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. When people want to show off code to each other they can put it on Github
+2. It helps to track different versions of the programme.
+3. It helps to work in team collaboratively
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+It creates a repository in the github
```

Comment:
When you fork a repo you are copying an existing repo so that you can edit it without making changes to the original, not just creating a new one